src/Exercism.Analyzers.CSharp/Analyzers/Leap/LeapSyntax.cs

```diff
@@ -35,6 +38,15 @@ internal static class LeapSyntax
                 .OfType<IfStatementSyntax>()
                 .Any();
 
+        public static bool UsesNestedIfStatement(this LeapSolution leapSolution) =>
+            leapSolution.IsLeapYearMethod
+                .DescendantNodes()
+                .OfType<IfStatementSyntax>()
+                .Any(_ => _.DescendantNodes()
```

Comment:
The `_` is usually used for things that are not used. Maybe you could use a more descriptive name here? Possible just `ifStatement`

Reply:
Sure, I changed it. 👍